docs/01-module-lesson1/00-structure.adoc

```diff
@@ -0,0 +1,20 @@
+// header file for curriculum section 1: ERSTE LERNEINHEIT
+// (c) iSAQB e.V. (https://isaqb.org)
```

Comment:
Folder name should be `01-module-lesson-one`, `02-module-lesson-two`, etc.

Reply:
done.